var/spack/repos/builtin/packages/margo/package.py

```diff
@@ -7,43 +7,49 @@
 
 
 class Margo(AutotoolsPackage):
-    """Argobots bindings to the Mercury RPC API"""
+    """A library that provides Argobots bindings to the Mercury RPC
+    implementation.  This name will be deprecated soon; please use the
+    mochi-margo package instead."""
 
-    homepage = "https://xgitlab.cels.anl.gov/sds/margo"
-    url      = "https://xgitlab.cels.anl.gov/sds/margo/-/archive/v0.4.3/margo-v0.4.3.tar.gz"
-    git      = "https://xgitlab.cels.anl.gov/sds/margo.git"
+    homepage = 'https://xgitlab.cels.anl.gov/sds/margo'
+    git = 'https://xgitlab.cels.anl.gov/sds/margo.git'
 
-    maintainers = ['fbudin69500', 'chuckatkins', 'carns']
+    maintainers = ['carns', 'mdorier']
 
+    version('master', branch='master')
     version('develop', branch='master')
-    version('0.4.3', sha256='61a634d6983bee2ffa06e1e2da4c541cb8f56ddd9dd9f8e04e8044fb38657475')
-
-    variant('shared', default=True,
-            description='Build shared libraries instead of static libraries')
-
-    depends_on('autoconf',  type='build')
-    depends_on('automake',  type='build')
-    depends_on('libtool',   type='build')
-    depends_on('m4',        type='build')
-    depends_on('pkgconfig', type='build')
-
-    depends_on('mercury')
+    version('0.9', tag='v0.9')
```

Comment:
Instead of using tags, which are moving targets in git, could we use URLs and checksums for all versions (for both packages)?